frontend/test/metabase/scenarios/dashboard/actions-on-dashboards.cy.spec.js

```diff
@@ -0,0 +1,136 @@
+import {
+  restore,
+  queryWritableDB,
+  resetTestTable,
+  getTableId,
+  fillActionQuery,
+  resyncDatabase,
+} from "__support__/e2e/helpers";
+
+import { WRITABLE_DB_ID } from "__support__/e2e/cypress_data";
+
+const TEST_TABLE = "scoreboard_actions";
+
+["mysql", "postgres"].forEach(dialect => {
+  describe(
+    `Write Actions on Dashboards (${dialect})`,
+    { tags: ["@external", "@actions"] },
+    () => {
+      beforeEach(() => {
+        cy.intercept("/api/card/*").as("getModel");
+
+        cy.intercept("POST", "/api/dashboard/*/dashcard/*/execute").as(
+          "executeAPI",
+        );
+
+        cy.intercept("GET", "/api/dashboard/*").as("dashboardLoad");
+
+        resetTestTable({ type: dialect, table: TEST_TABLE });
+        restore(`${dialect}-writable`);
+        cy.signInAsAdmin();
+        resyncDatabase(WRITABLE_DB_ID);
+      });
+
+      it("should show writable_db with actions enabled", () => {
+        cy.visit(`/admin/databases/${WRITABLE_DB_ID}`);
+        cy.get("#model-actions-toggle").should("be.checked");
+      });
```

Comment:
We've already covered this, right?
I don't see the benefit of repeating it here.